tests/installation/yast2_firstboot.pm

```diff
@@ -127,6 +118,10 @@ sub run {
         my $client_method = \&{"$client"};
         $client_method->($self, $custom_needle);
     }
+    # Compare the existing settings with the expected ones as specified in test_data
```

Comment:
it would be less confusing call them, pre-selected values or default values. At least for me settings is more wide meaning. wdyt?

Reply:
the current comparison concerns only pres elected values, but we don't know if in the future we might want to compare e.g. the typed username with the one set in test_data